core/src/avm2/object/event_object.rs

```diff
@@ -10,6 +10,7 @@ use crate::context::UpdateContext;
 use crate::display_object::TDisplayObject;
 use crate::display_object::{DisplayObject, InteractiveObject, TInteractiveObject};
 use crate::events::{KeyCode, MouseButton};
+use crate::prelude::Avm2Value;
```

Comment:
We already import `Value` here from `avm2::value::Value`; this is unnecessary

Reply:
Right!